lectures/greek_square.md

```diff
@@ -137,30 +172,36 @@ If we choose $(y_{-1}, y_{-2})$ to set $(\eta_1, \eta_2) = (1, 0)$, then $y_t =
 
 If we choose $(y_{-1}, y_{-2})$ to set $(\eta_1, \eta_2) = (0, 1)$, then $y_t = \delta_2^t$ for all $t \geq 0$.
 
+Soon we'll relate the preceding calculations to components an eigen decomposition of a transition
```

Comment:
@HumphreyYang this sentence isn't clear to me. Should we add `of an eigen decompostion to a transition matrix"?

Reply:
Hi @mmcky, I think the sentence is split into two lines and the line below writes exactly what you said. I will merge these two lines.